### ISSUE TITLE
ground clock in (SMA) and out (MMCX) [v1.1 BOM change]

### ISSUE THREAD

**Issue by jordens** [sinara-hw/Kasli#26]
I have a couple times now that that floating shield thing is a bad idea, also on crate-internal MMCX wiring. The symptom is 400/800 kHz switcher modulation sidebands on the urukul clock. Shorting the shield on both ends resolved it each time.
This can a pure BOM change and I'd like to have it implemented on all future Kasli v1.1 builds and Kasli v2.0 as well. 

c.f. #5 and maybe #10

**Comment by hartytp** [comment]
Agreed. Floating grounds can be the right thing to do in rare cases, but it's not a sensible default option.

**Comment by hartytp** [comment]
(I thought we agreed to do this ages ago)

**Comment by jordens** [comment]
We agreed. I'm just pushing to rush this into the current boards without waiting for v1.2/v2.0.

**Comment by marmeladapk** [comment]
a87c75c

**Comment by jordens** [comment]
Thanks. The v1.1.1 version you assigned is a very good idea.
